jobs/job_runner.go

```diff
@@ -162,3 +173,17 @@ var processPaidSessionsHandler = func(args worker.Args) error {
 
 	return nil
 }
+
+var claimUnusedPRLsHandler = func(args worker.Args) error {
+	thresholdTime := time.Now().Add(-3 * time.Hour) // consider a transaction timed out if it takes more than 3 hours
```

Comment:
We can change this timed out time (3 hours) if we want but I know sometimes it takes a long time if the network is congested.